### ISSUE TITLE
Remove core.exportVariable the console log info 

### ISSUE THREAD

**Issue by xyngfei** [actions/toolkit#1301]
When I use the following code, it will auto-print the info into the terminal.
`core.exportVariable(secretNamePOSIX, secretValue);`

<img width="145" alt="image" src="https://user-images.githubusercontent.com/17619539/210500372-64d1b332-a1d6-4154-a36e-dcc523a67427.png">

Does it have any feature flag, no to print out the info into the terminal?

**Comment by navtoj** [comment]
@xyngfei is this what you're looking for? https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#masking-a-value-in-log

**Comment by xyngfei** [comment]
@navtoj no, even with the code below, i am still getting console log info when I export the variable

```
core.setSecret(secretNamePOSIX);
core.exportVariable(secretNamePOSIX, secretValue);
```
